va-gov/pages/disability-benefits/after-you-apply.md

```diff
@@ -12,7 +12,7 @@ majorlinks:
     links:
     - url: /disability-benefits/after-you-apply/date/
       title: Effective Date of Disability
-      description: "Find out when you’ll start getting your disability payments."
+      description: "Fiwhen you’ll start getting your disability payments."
```

Comment:
@bethpotts is this how you want it. Doesn't look right.

Reply:
@accbjt You're right, thank you for noticing. And I found one more with a weird typo. This is part of what we'd catch in our last look at the titles that we still need to do in dev-preview. You'll still hold on merging this to preview, right?

Here's what should appear on these two pages. Can you fix these for me?:

va-gov/pages/disability-benefits/after-you-apply.md
Title: The VA Claim Process After You File Your Claim

va-gov/pages/disability-benefits/after-you-apply/date/ 
(TITLE AND DESCRIPTION BOTH NEED EDITS)
Title: Disability Compensation Effective Date
Description: "Find out when you’ll start getting your disability payments."